File: parchment routes and obelisks/midleway/obelisk_system.java
public class Obelisk {
    private int health;
    private int damage;
    private boolean isBaseObelisk;

    public Obelisk(boolean isBase) {
        this.isBaseObelisk = isBase;
        this.health = isBase ? 2000 : 1200;
        this.damage = isBase ? 200 : 120;
    }

    public void attack() {
        System.out.println("Obelisk fires a magical beam!");
    }
}

File: parchment routes and obelisks/highland/obelisk_system.py
class Obelisk:
    def __init__(self, is_base=False):
        self.is_base = is_base
        self.health = 2000 if is_base else 1200
        self.damage = 200 if is_base else 120

    def attack(self):
        print("Obelisk fires a magical beam!")
